doc/plugins/fault-injection-cn.md

```diff
@@ -0,0 +1,139 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+[English](fault-injection.md)
+
+# fault-injection
+
+故障注入插件，该插件可以和其他插件一起使用，并且会在其他插件前被执行，配置 `abort` 参数将直接返回给客户端指定的响应码并且终止其他插件的执行，配置 `delay` 参数将延迟某个请求，并且还会执行配置的其他插件。
+
+### 参数
+
+|名称    |必须|描述|
+|------- |-----|------|
+|abort.http_status|是|返回给客户端的 http 状态码|
```

Comment:
I think it can be set a default value 200 if the user not specified.

Reply:
I don't think so. This plugin is for `fault` injection, `200` is not fault return.

Reply:
If the user specify neither http_status nor body, seems that the readability is not good.